runtime-builder/tests/integration/go-test.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+- name: 'gcr.io/${PROJECT_ID}/go1-builder:staging'
```

Comment:
the tag on this builder image will actually be templated (so that we can pass in the tag through a kokoro build param), but i'll change that in a later PR when I add that change to the integration test framework.